### PR TITLE
Multiple tables

### PR DESCRIPTION
**Pull request by Brenda309**
### Project requirements
Create a table named owners with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
full_name: string
age: integer
Create a table named species with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
Modify animals table:
Make sure that id is set as autoincremented PRIMARY KEY
Remove column species
Add column species_id which is a foreign key referencing species table
Add column owner_id which is a foreign key referencing the owners table
Remember all this goes in schema.sql.
**Insert the following data into the owners' table:**
Sam Smith 34 years old.
Jennifer Orwell 19 years old.
Bob 45 years old.
Melody Pond 77 years old.
Dean Winchester 14 years old.
Jodie Whittaker 38 years old.
Insert the following data into the species table:
Pokemon
Digimon
**Modify your inserted animals so it includes the species_id value:**
If the name ends in "mon" it will be Digimon
All other animals are Pokemon
**Modify your inserted animals to include owner information (owner_id):**
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarman.
Remember these insertions and modifications happen in data.sql.
**Write queries (using JOIN) to answer the following questions:**
**What animals belong to Melody Pond?**
querie_for_MP_animals

**List of all animals that are pokemon (their type is Pokemon).**
pokemon_species2

List all owners and their animals, remember to include those that don't own any animal.
owners_animals3

How many animals are there per species?
total_animal_gr_by_sp

List all Digimon is owned by Jennifer Orwell.
Owned_by_JO

List all animals owned by Dean Winchester that haven't tried to escape.
Escape_attempt
Who owns more animals
user_with_more_anim

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -14,6 +15,120 @@ INSERT INTO Animals(name, date_of_birth, neutered, weight_kg, escape_attempts) V
...
+
+/*Modify your inserted animals to include owner information (owner_id)*/
+UPDATE Animals
+SET  owners_id = 1
+WHERE name =  'Agumon';
+
+UPDATE Animals
+SET  owners_id = 3
+WHERE name = 'Devimon' ;
+
```
Comment (BenMukebo) [inline data.sql:108]: - [ ] Great job so far 👏🏻 however you're missing on refreshments in your query:
- Modify your inserted animals to include owner information (owner_id): Jennifer Orwell owns Gabumon and Pikachu. Kindly add this 👍🏻 
This should be like this:

```sql
 UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';
```

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi @Brenda309,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- Gitflow was correctly used.  ✔️   
- Good commit messages ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @Brenda309.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions are implemented 👍 

### Optional suggestions

- Please, consider fixing the broken link to the MIT.md file 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Brenda309** [comment]
Hey @elmar8287, Nice catch on MIT Link:rofl: 
I will definitely fix it. meanwhile not that I can't figure it out but if you have a working link do you mind dropping it here?  because most of those that I have they are all seem to be broken. Thank you again for catching that.

**Pull request merged**